microsoft-365/security/defender-endpoint/linux-whatsnew.md

```diff
@@ -30,6 +30,57 @@ This article is updated frequently to let you know what's new in the latest rele
 - [What's new in Defender for Endpoint on macOS](mac-whatsnew.md)
 - [What's new in Defender for Endpoint on iOS](ios-whatsnew.md)
 
+<details>
+	<summary> April-2023 (Build: 101.98.40 | Release version: 30.123022.19840.0)</summary>
+
+&ensp;Released: **April , 08,2023**<br/>
+&ensp;Published: **April 08, 2023**<br/>
+&ensp;Build: **101.98.40**<br/>
+&ensp;Release version: **30.123022.19840.0**<br/>
+&ensp;Engine version: **1.1.20000.2**<br/>
+&ensp;Signature version: **1.381.3067.0**<br/>
+
+**What's new**
+
+- There are multiple fixes and new changes in this release 
+	- Logging and error reporting improvements for auditd.
+	- Handle failure in reload of auditd configuration.
+	- Handling for empty auditd rule file during MDE install.
+	- Engine Update to 1.1.20000.2 and Signatures Ver: 1.381.3067.0.
+	- Other fixes
+	
+**Known issues**
+
+- While upgrading mdatp to version 101.94.13, you may notice that health is false, with health_issues as "no active supplementary event provider". This may happen due to misconfigured/conflicting auditd rules on existing machines. To mitigate the issue, the auditd rules on the existing machines need to be fixed. The following steps can help you to identify such auditd rules (these commands need to be run as super user). Please take backup of following file: /etc/audit/rules.d/audit.rules as these steps are only to identify failures.
```

Comment:
```suggestion
- While upgrading mdatp to version 101.94.13, you may notice that health is false, with health_issues as "no active supplementary event provider". This may happen due to misconfigured or conflicting audit rules on existing devices. To mitigate the issue, the audit rules on the existing devices need to be fixed. The following steps can help you identify such audit rules (these commands should be run as super user). Make a backup of the following file: /etc/audit/rules.d/audit.rules, as these steps only identify failures.
```